perceval/utils/qmath.py

```diff
@@ -61,3 +64,48 @@ def exponentiation_by_squaring(base, power: int):
         temp_base = temp_base * temp_base
 
     return result
+
+
+def kmeans(features: np.ndarray, n_clusters: int, n_init: int = 10) -> np.ndarray:
+    """
+    Manual KMeans implementation.
+
+    :param features: Data points for clustering.
```

Comment:
parameter type is already written in the function declaration (L69). Remove the `:type` from docstring. Could you please do this for all other methods too wherever `:type` is written in docstrings.